### DISCUSSION TITLE
Full Stack Example on AWS lambda?

### DISCUSSION THREAD

**Discussion by ajaishankar** [honojs/hono#2648] [General]
Hi

The Hono [AWS lambda](https://hono.dev/getting-started/aws-lambda) shows an example of API endpoint.

Is there a example of running a full stack application on lambda, similar to CloudFlare Pages?

The [Remix Grunge Stack](https://github.com/remix-run/grunge-stack/blob/main/app.arc) sample deploys the whole app as a lambda along with static assets using [Architect](https://arc.codes/docs/en/get-started/quickstart)

How would one go about setting up something similar with Hono?

Help much appreciated!

Thanks

Ajai

**Comment by yusukebe** [comment]
Hi @ajaishankar 

> Is there a example of running a full stack application on lambda, similar to CloudFlare Pages?

No.

> How would one go about setting up something similar with Hono?

Good. You or anyone can do it. But it's difficult to put it on [honojs/examples](https://github.com/honojs/examples) because maintaining the examples is sometimes hard for us. I think the best solution is for someone to create their own project on GitHub and make the Hono website link to it.

**Reply by ajaishankar** [reply]
Got it, thanks! Will give this a shot...
